packages/synapse-react-client/src/components/PortalList/PortalCard.tsx

```diff
@@ -0,0 +1,182 @@
+                    size="small"
+                    onClick={() => setIsAclEditorOpen(true)}
+                    aria-label="Manage portal permissions"
+                    title="Manage portal permissions"
+                  >
+                    <Settings fontSize="small" />
+                  </IconButton>
+                )}
+                {showDeleteButton && (
+                  <IconButton
+                    size="small"
+                    onClick={() => setIsDeleteDialogOpen(true)}
+                    aria-label="Delete portal"
+                    title="Delete portal"
+                    color="error"
+                  >
+                    <Delete fontSize="small" />
+                  </IconButton>
+                )}
+              </Box>
+            </Box>
+            {url && (
+              <MuiLink
+                href={url}
+                target="_blank"
+                rel="noopener noreferrer"
+                className={styles.url}
+              >
+                {url}
+              </MuiLink>
+            )}
+          </Box>
+
+          <Box className={styles.metadata}>
+            {id && (
+              <Typography variant="body2" color="text.secondary">
+                <strong>ID:</strong> {id}
+              </Typography>
+            )}
+
+            {createdBy && (
+              <Box className={styles.userInfo}>
+                <Typography variant="body2" color="text.secondary">
+                  <strong>Created by:</strong>
+                </Typography>
+                <UserCard
+                  ownerId={createdBy}
+                  size="SMALL USER CARD"
+                  hideEmail
+                  disableLink={false}
+                />
+                {createdOn && (
+                  <Typography variant="caption" color="text.secondary">
+                    on {dayjs(createdOn).format('MMMM D, YYYY')}
+                  </Typography>
+                )}
+              </Box>
+            )}
+
+            {modifiedBy && (
+              <Box className={styles.userInfo}>
+                <Typography variant="body2" color="text.secondary">
+                  <strong>Last modified by:</strong>
+                </Typography>
+                <UserCard
+                  ownerId={modifiedBy}
+                  size="SMALL USER CARD"
+                  hideEmail
+                  disableLink={false}
+                />
+                {modifiedOn && (
+                  <Typography variant="caption" color="text.secondary">
+                    on {dayjs(modifiedOn).format('MMMM D, YYYY')}
+                  </Typography>
+                )}
+              </Box>
+            )}
+          </Box>
+        </CardContent>
+      </Card>
+
+      {showAclEditor && id && (
+        <PortalAclEditorModal
+          portalId={id}
+          open={isAclEditorOpen}
+          onClose={() => setIsAclEditorOpen(false)}
+        />
+      )}
+
+      {id && (
+        <ConfirmationDialog
```

Comment:
Surfacing delete kind of worries me. Hopefully the server restricts deleting portals which have DOIs 😉